isamples_metadata/GEOMETransformer.py

```diff
@@ -0,0 +1,362 @@
+                elif "|" in collector_list:
+                    for collector in collector_list.split("|"):
+                        responsibilities_pieces.append(f"collector:{collector}")
+                else:
+                    responsibilities_pieces.append(f"collector:{collector_list}")
+            principal_investigator = parent_record.get("principalInvestigator")
+            if principal_investigator is not None:
+                responsibilities_pieces.append(
+                    f"principalInvestigator:{principal_investigator}"
+                )
+            identified_by = parent_record.get("identifiedBy")
+            if identified_by is not None:
+                responsibilities_pieces.append(f"identifiedBy:{identified_by}")
+            tax_team = parent_record.get("taxTeam")
+            if tax_team is not None:
+                responsibilities_pieces.append(f"taxonomy team:{tax_team}")
+            entered_by = parent_record.get("eventEnteredBy")
+            if entered_by is not None:
+                responsibilities_pieces.append(f"event registrant:{entered_by}")
+            return responsibilities_pieces
+        return []
+
+    def produced_by_result_time(self) -> typing.AnyStr:
+        parent_record = self._source_record_parent_record()
+        if parent_record is not None:
+            result_time_pieces = []
+            year = parent_record.get("yearCollected")
+            if year is not None:
+                result_time_pieces.append(year)
+            month = parent_record.get("monthCollected")
+            if month is not None:
+                month = month.zfill(2)
+                result_time_pieces.append(month)
+            day = parent_record.get("dayCollected")
+            if day is not None:
+                day = day.zfill(2)
+                result_time_pieces.append(day)
+            return "-".join(result_time_pieces)
+        return Transformer.NOT_PROVIDED
+
+    def sampling_site_description(self) -> typing.AnyStr:
+        parent_record = self._source_record_parent_record()
+        if parent_record is not None:
+            habitat = parent_record.get("habitat")
+            if habitat is not None:
+                return habitat
+            else:
+                depth_to_bottom = parent_record.get("depthOfBottomInMeters")
+                if depth_to_bottom is not None:
+                    return f"Depth to bottom {depth_to_bottom} m"
+        return Transformer.NOT_PROVIDED
+
+    def sampling_site_label(self) -> typing.AnyStr:
+        parent_record = self._source_record_parent_record()
+        if parent_record is not None:
+            return parent_record.get("locality", Transformer.NOT_PROVIDED)
+        return Transformer.NOT_PROVIDED
+
+    def sampling_site_elevation(self) -> typing.AnyStr:
+        # Note that this is subject to revision based on the outcome of
+        # https://github.com/isamplesorg/metadata/issues/35
```

Comment:
I'll need to clean this up based on the notes in that issue.  Will do in a subsequent revision.